javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/JavaParserFacade.java

```diff
@@ -266,27 +263,27 @@ private void solveArguments(Node node, NodeList<Expression> args, boolean solveL
 
     /**
      * Get the type associated with the node.
-     *
+     * <p>
      * This method was originally intended to get the type of a value: any value has a type.
-     *
+     * <p>
      * For example:
      * <code>
      * int foo(int a) {
-     *         return a; // when getType is invoked on "a" it returns the type "int"
-     *     }
+     * return a; // when getType is invoked on "a" it returns the type "int"
```

Comment:
no idea how to put space into the tag, maybe `&nbsp;`?

Reply:
Using <pre> instead of <code> helps.